commit eeddfa5be17ba0fa2d98874aa881683373bf7180
Author: Kees Cook <kees@kernel.org>
Date:   Thu Dec 14 16:13:54 2023 -0800

    docs: conf.py: Ignore __counted_by attribute
    
    It seems that Sphinx is confused by the __counted_by attribute on struct
    members. Add it to the list of known attributes.
    
    Reported-by: kernel test robot <lkp@intel.com>
    Closes: https://lore.kernel.org/oe-kbuild-all/202312150614.kOx8xUkr-lkp@intel.com/
    Cc: Jonathan Corbet <corbet@lwn.net>
    Cc: "Gustavo A. R. Silva" <gustavoars@kernel.org>
    Cc: linux-doc@vger.kernel.org
    Cc: linux-hardening@vger.kernel.org
    Signed-off-by: Kees Cook <keescook@chromium.org>
    Signed-off-by: Jonathan Corbet <corbet@lwn.net>
    Link: https://lore.kernel.org/r/20231215001347.work.151-kees@kernel.org

diff --git a/Documentation/conf.py b/Documentation/conf.py
index 3a1a804c3a13..a3d1fe9dfcf9 100644
--- a/Documentation/conf.py
+++ b/Documentation/conf.py
@@ -106,6 +106,7 @@ if major >= 3:
             "__weak",
             "noinline",
             "__fix_address",
+            "__counted_by",
 
             # include/linux/memblock.h:
             "__init_memblock",